packages/app/src/app/overmind/namespaces/editor/internalActions.ts

```diff
@@ -72,7 +72,7 @@ export const initializeLiveSandbox: AsyncAction<Sandbox> = async (
       // to join
       state.live.joinSource === 'sandbox'
     ) {
-      await actions.live.internal.disconnect();
```

Comment:
Sure you do not want to wait until the disconnect is done?

Reply:
The function doesn't return a promise, so this is a bit misleading.